pandas/core/arrays/datetimes.py

```diff
@@ -324,6 +430,26 @@ def _generate_range(cls, start, end, periods, freq, tz=None,
 
         return cls._simple_new(index.asi8, freq=freq, tz=tz)
 
+    # -----------------------------------------------------------------
+    # DatetimeLike Interface
+
+    def _unbox_scalar(self, value):
+        if not isinstance(value, (self._scalar_type, type(NaT))):
+            raise ValueError("'value' should be a a Timestamp..")
+        return value.value
+
+    def _scalar_from_string(self, value):
```

Comment:
are these doc-stringed in the super class?

Reply:
Yeah (this one was missing one. added.)